docs/en/stack/ml/df-analytics/ml-dfa-outlier-detection.asciidoc

```diff
@@ -8,28 +8,22 @@
 find unusual data points in a data set compared to the normal data points.
 
 This page contains a conceptual overview of {oldetection} in the {stack} and a 
-simple <<ecommerce-outliers,example>> of how to use it.
+simple <<weblogs-outliers,example>> of how to use it.
 
 
 [discrete]
 [[dfa-outlier-detection]]
 == {oldetection-cap}
 
-{oldetection-cap} is an analysis for identifying data points (outliers) whose 
-feature values are different from those of the normal data points in a 
-particular data set. Outliers may denote errors or unusual behavior. 
-{oldetection-cap}, for example, can be used to detect malicious software on a 
-machine, unusual user behavior on a network, or any unusual entity in a given 
-population. As {oldetection} algorithms operate on the assumption that the 
-outliers make up a small proportion of the overall data population. 
+{oldetection-cap} is identification of data points that are significantly 
+different from other values in the data set. For example, outliers could be 
+errors or unusual entities in a data set. {oldetection-cap} is an unsupervised 
+{ml} technique, there is no need to provide training data.
 
-The {ml-features} provide unsupervised {oldetection}, which means there is no 
-need to provide a training data set. Unsupervised {oldetection} uses various 
-machine learning techniques to find which data points are unusual compared to 
-the majority of the data points.
+IMPORTANT: {oldetection-cap} is a batch analysis, it runs against your data 
+once. If new data comes into the index, you need to do the analysis again on the 
+altered data.
 
-You can create {oldetection} {dfanalytics-jobs} in {kib} or by using the
-{ref}/put-dfanalytics.html[create {dfanalytics-jobs} API].
 
 [discrete]
 [[dfa-outlier-algorithms]]
```

Comment:
> === {oldetection-cap} algorithms

If you get rid of the outlier detection section title, I'd suggest bumping this one (and all the others like this) up to a heading level 2 now.